xarray/core/dataset.py

```diff
@@ -1161,13 +1162,16 @@ def sel(self, method=None, tolerance=None, drop=False, **indexers):
         return result._replace_indexes(new_indexes)
 
     def isel_points(self, dim='points', **indexers):
+        # type: (...) -> Dataset
         """Returns a new dataset with each array indexed pointwise along the
         specified dimension(s).
 
         This method selects pointwise values from each array and is akin to
         the NumPy indexing behavior of `arr[[0, 1], [0, 1]]`, except this
         method does not require knowing the order of each array's dimensions.
 
+        Will use dask vectorised operation if available
```

Comment:
Let's not add this comment -- implementation details usually don't make it into docstrings